### ISSUE TITLE
Next on attribute

### ISSUE THREAD

**Issue by M-See** [silviokennecke/ha-custom-components#9]
Hi mate, thanks for your nice Lovelace card.

I see that the "next on" attribute is falsely accounted as the arrival time for the "next" connection. But it´s actually the departure time of the next-next connection.

But if you find a way to extract the correct departure times from each connection from the "departures" attribute (depending on the number of connections specified of course) you could (hopefully) change this to make it right an nicely working.

Apart from that already looking nice :)!
Looking forward to those enhancements :)!

**Comment by M-See** [comment]
With the fix of my pull-request ([Pull](https://github.com/FaserF/ha-deutschebahn/pull/28)

you could fix this by changing the attributes to:

```  next_departure_time: next_1
  next_arrival_time: next_1_arrival```
  
  But maybe more connection would be nice to show as well.

**Comment by silviokennecke** [comment]
I’ve just updated the documentation. I’ll also like your idea to add the possibility of showing more than only the next connection. I‘m currently very busy again, so I cannot estimate, when I will be able to implement this (if not already done in a PR when I find time for this)

**Comment by silviokennecke** [comment]
I just released version 1.1.0, which allows you to use the departures attribute of the ha-deutschebahn sensor. You can find a sample configuration in the wiki (https://github.com/silviokennecke/ha-custom-components/wiki/Components#array-of-connections-in-one-attribute)
